algorithms/gr-bfsrh/cpp/BFS.cpp

```diff
@@ -0,0 +1,87 @@
+#include <iostream>
+#include <list>
+
+using namespace std;
+
+class Graph {
+  int numVertices;
+  list<int>* adjLists;
+  bool* visited;
+
+   public:
+  Graph(int vertices);

```

Comment:
Codacy found an issue: [Class 'Graph' has a constructor with 1 argument that is not explicit.](https://app.codacy.com/gh/FOSS-UCSC/FOSSALGO/pullRequest?prid=6318755)